### PR TITLE
added the logic for admin users

### PR DESCRIPTION
**Pull request by Alexander7337**
added admin list with user who have all cards/documents on dashboard

### CODE REVIEW

File: src/app/cards/cards.component.ts [hunk 1]
```diff
@@ -212,6 +215,12 @@ export class CardsComponent implements OnInit, OnDestroy {
       );
     });
 
+    this._piaService.availableAdmins.forEach(admin => {
```
Comment (ennedigi) [inline src/app/cards/cards.component.ts]: if you want to do this, do it in the then block above as the block below is asyncrounous, so at this line you're not sure that the availableAdmins array is already populated

**Review by ennedigi — COMMENTED** [review]
1 major comment,

File: src/app/cards/cards.component.ts (hunk 1 above)
Comment (Alexander7337) [reply src/app/cards/cards.component.ts]: availableAdmins array is populated on NgOnInit(). I guess this is the first thing which is done after the component is created. 
I tried the thing you suggested and placed this function inside them just after the availableAdmins is populated but the result at the end is NOT expected.

Comment (Alexander7337) [reply src/app/cards/cards.component.ts]: In addition, the same rules apply for the previous part of this block, where we return cards for the involved users only.

Comment (ennedigi) [reply src/app/cards/cards.component.ts]: OK

**Pull request merged**